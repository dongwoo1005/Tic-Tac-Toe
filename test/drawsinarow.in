game stdin stdin
C
W
NW
SE
S
N
SW
NE
E
game stdin stdin
C
W
NW
SE
S
N
SW
NE
E
quit
