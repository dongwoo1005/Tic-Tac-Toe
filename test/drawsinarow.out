X's move
O's move
X's move
O's move
X's move
O's move
X's move
O's move
X's move
Draw
Score is
X 0
O 0
O's move
X's move
O's move
X's move
O's move
X's move
O's move
X's move
O's move
Draw
Score is
X 0
O 0
